### PR TITLE
game landing page jsx and css

### PR DESCRIPTION
**Pull request by ritamwaikinda**

### CODE REVIEW

File: client/src/App.css [hunk 1]
```diff
@@ -9,6 +9,9 @@
 
 body {
   display: flex;
+  background-image: url(https://i.imgur.com/SrYnZDZ.gif?noredirect);
```
Comment (ErnieAtLYD) [inline client/src/App.css:12]: Hi @ritamwaikinda, I would make sure the URLs are nested in single quotes, like this:  `background-image: url('https://i.imgur.com/SrYnZDZ.gif?noredirect');`

**Review by ErnieAtLYD — CHANGES_REQUESTED** [review]
Hi Rita - I see a file for `package-lock.json`  which is a big no-no, as I'm assuming that file was generated with `npm` - chances are the file should be a `yarn.lock` which is the file created when using `yarn`. 

My recommendation is that you delete the `package-lock.json` file and re-run `yarn`, checking in any update yarn.lock files as necessary.

Let myself or a TA know if that doesn't make sense!

**Comment by ritamwaikinda** [comment]
Ernie, thank you so much!! Yes, the yarn thing makes sense and I appreciate
you letting me know, I had no idea!! It's awesome now that all this is
starting to make a little sense :D

**Comment by saritasalazar** [comment]
New pull request was created closing this one.

**Pull request closed without merging**